pkg/storage/engine/rocksdb.go

```diff
@@ -367,13 +367,13 @@ func (r *RocksDB) open() error {
 
 	status := C.DBOpen(&r.rdb, goToCSlice([]byte(r.dir)),
 		C.DBOptions{
-			cache:           r.cache.cache,
-			block_size:      C.uint64_t(blockSize),
-			wal_ttl_seconds: C.uint64_t(walTTL),
-			allow_os_buffer: C.bool(true),
-			logging_enabled: C.bool(log.V(3)),
-			num_cpu:         C.int(runtime.NumCPU()),
-			max_open_files:  C.int(r.maxOpenFiles),
+			cache:            r.cache.cache,
+			block_size:       C.uint64_t(blockSize),
+			wal_ttl_seconds:  C.uint64_t(walTTL),
+			use_direct_reads: C.bool(false),
```

Comment:
There isn't a strong need to for this setting, but there is a need for `use_direct_writes`. Can you follow up with a PR which replaces this with `use_direct_writes`?